lib/gilmour/backends/backend.rb

```diff
@@ -61,6 +107,16 @@ def publish(message, destination, opts = {}, code = 0, &blk)
       sender
     end
 
+    def emit_error(message)
```

Comment:
queue vs publish is a redis specific thing. emit_error implementation should directly go into redis backend